test/integration/cart_integration_test.rb

```diff
@@ -104,6 +104,75 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     assert_equal showcart_path, current_path
   end
 
+  test 'a user can edit the quantity of an item in their cart' do
+    coffee = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    aeropress = Item.create(title: 'aeropress', description: 'light stuff', price: 1300)
+    visit "/items/#{coffee.id}"
+    click_link_or_button('Add to Cart')
+    visit "/items/#{aeropress.id}"
+    click_link_or_button('Add to Cart')
+    click_link_or_button('Cart')
+
+    within("#item_#{coffee.id}") do
+      select '7', from: 'update_item_quantity[quantity]'
+      click_link_or_button('Update Quantity')
+    end
+
+    within('#flash_notice') do
+      assert page.has_content?('Item quantity updated')
+    end
+    assert page.has_content?('coffee')
+    within("#quantity_#{coffee.id}") do
+      assert page.has_content?('7')
+    end
+    assert page.has_content?('aeropress')
+    assert_equal showcart_path, current_path
+  end
+
+  test 'a user can delete an item from their cart' do
+    coffee = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    aeropress = Item.create(title: 'aeropress', description: 'light stuff', price: 1300)
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.